selftests/pre_release/jobs/timesensitive.py

```diff
@@ -14,11 +15,13 @@
     'run.references': [os.path.join(ROOT_DIR, 'selftests', 'unit'),
                        os.path.join(ROOT_DIR, 'selftests', 'functional')],
     'filter.by_tags.tags': ['parallel:1'],
+    'nrunner.status_server_uri': '127.0.0.1:9999',
```

Comment:
How about using the default 8888 used elsewhere? Just a bit less entropy in the code this way as all included values are the same.

Reply:
Good catch, yes, the examples are using 9999 but the default is 8888. will change.

Reply:
> How about using the default 8888 used elsewhere? Just a bit less entropy in the code this way as all included values are the same.

+1.  But the fact is, there's some smelly here.  It should not be necessary to set a status server given that it's registered.  I've created an issue to track it here: https://github.com/avocado-framework/avocado/issues/4157